esp/templates/program/modules/finaidapprovemodule/finaid.html

```diff
@@ -0,0 +1,71 @@
+{% extends "main.html" %}
+
+{% block title %}Approve Financial Aid Requests for {{program.niceName}}{% endblock %}
+
+{% block stylesheets %}
+    {{ block.super }}
+    <link rel="stylesheet" type="text/css" href="/media/styles/forms.css" />
+    <style>
+    #program_form {
+      overflow: auto;
+      max-height: 500px;
+      width: 700px;
+      margin: 0px 10px 15px 10px;
+    }
+    </style>
+{% endblock %}
+
+{% block xtrajs %}
+<script type="text/javascript" language="javascript">
+function toggle(source) {
+  checkboxes = document.getElementsByName('user');
```

Comment:
You should be able to do all of this with a single line of jQuery:
`$j('input[name=user]').prop('checked', source.checked);`